frontend/WikiContrib-Frontend/src/query.js

```diff
@@ -749,64 +749,77 @@ export class Query extends Component {
-                                        />
+                                      <input
+                                        className="user_input"
+                                        value={obj.fullname}
+                                        name="fullname"
+                                        id="fullname"
+                                        required
+                                        onChange={e =>
+                                          this.handlChange(
+                                            e.target.name,
+                                            e.target.value,
+                                            index
+                                          )
+                                        }
+                                      />
+                                      <label className="user_label" htmlFor="fullname">
+                                        Full name
+                                      </label>
                                       </Table.Cell>
                                       <Table.Cell
-                                        style={{ textAlign: 'center' }}
+                                      className="user_table_cell"
                                       >
-                                          <input
-                                            className="user_input"
-                                            value={obj.gerrit_username}
-                                            name="gerrit_username"
-                                            placeholder="Gerrit username"
-                                            onChange={e =>
-                                              this.handlChange(
-                                                e.target.name,
-                                                e.target.value,
-                                                index
-                                              )
-                                            }
-                                          />
+                                      <input
+                                        className="user_input"
+                                        value={obj.gerrit_username}
+                                        name="gerrit_username"
+                                        id="gerrit_username"
+                                        required
+                                        onChange={e =>
+                                          this.handlChange(
+                                            e.target.name,
+                                            e.target.value,
+                                            index
+                                          )
+                                        }
+                                      />
+                                      <label className="user_label" htmlFor="gerrit_username">
+                                        Gerrit Username
```

Comment:
U small in `Username`

Reply:
Ok @srish . I will make the changes and push